src/UserSpaceInstrumentation/CMakeLists.txt

```diff
@@ -56,7 +56,14 @@ target_link_libraries(UserSpaceInstrumentation PUBLIC
 # executed on function entry/exit of an instrumented function.
 add_library(OrbitUserSpaceInstrumentation SHARED)
 
-set_target_properties(OrbitUserSpaceInstrumentation PROPERTIES OUTPUT_NAME "orbituserspaceinstrumentation")
+# The linker flags mark all symbols coming from libraries as private and ensure that they will be hidden in the shared library.
+# The visibility properties ensure that symbols coming from this module's source files are considered private unless
+# explicitly marked visible in the source code file.
```

Comment:
In my opinion what is missing in this comment is _why_ we need the symbols to be hidden/private (it's explained in the commit message but I think it should be explain here as well).

Reply:
Good point. Done. Let me know what you think.